apps/full-site-editing/tsconfig.json

```diff
@@ -27,12 +27,16 @@
 		"esModuleInterop": false,
 
 		// Some type helpers from our webpack/
-		"types": [ "fse" ],
+		"types": [ "fse", "jest" ],
 		"typeRoots": [ "./typings", "./node_modules", "../../node_modules" ],
 
 		// Preserve JSX, allows compatibility with wp-element jsx babel transforms
 		"jsx": "preserve"
 	},
 	"exclude": [ "node_modules" ],
+	// Had to add that in order for vscode to recognize the tests and apply the jest types
+	"include": [
+		"*.test.ts",
+	],
```

Comment:
With `typeRoots` fixed, this can be removed.
```suggestion
```

Reply:
Yep, just did this. Thanks!